cpu/stm32_common/stm32_mem_lengths.mk

```diff
@@ -0,0 +1,308 @@
+# Set the common memory addresses for stm32 MCU family
+ROM_START_ADDR ?= 0x08000000
+RAM_START_ADDR ?= 0x20000000
+
+# The next block takes care of setting the rigth lengths of RAM and ROM
+# for the stm32 family. Most of the CPUs should have been taken into 
+# account here, so no need to assign the lengths per model.
+STM32_INFO := $(shell printf '%s' $(CPU_MODEL) | tr 'a-z' 'A-Z' | sed -E -e 's/^STM32(F|L)(0|1|2|3|4|7)([0-9])([0-9])(.)(.)/\1 \2 \2\3\4 \3 \4 \5 \6/')
+STM32_TYPE      = $(word 1, $(STM32_INFO))
+STM32_FAMILY    = $(word 2, $(STM32_INFO))
+STM32_MODEL     = $(word 3, $(STM32_INFO))
+STM32_MODEL2    = $(word 4, $(STM32_INFO))
+STM32_MODEL3    = $(word 5, $(STM32_INFO))
+STM32_PINCOUNT  = $(word 6, $(STM32_INFO))
+STM32_ROMSIZE   = $(word 7, $(STM32_INFO))
+
+ifeq ($(STM32_TYPE), F)
+  ifeq ($(STM32_FAMILY), 0)
+    ifeq ($(STM32_MODEL2), 3)
+      ifneq (, $(filter $(STM32_MODEL3), 0))
```

Comment:
Please remove all the `filter` calls where neither operand is a list, such as this line

Reply:
Ping! This was not addressed

Reply:
Oh! true, sorry, I'll fix it asap.

Reply:
Addressed.